src/utils/models.py

```diff
@@ -457,6 +457,7 @@ def get_records(self):
         anything new. If the previous import failed or there is new data,
         the import is marked as ongoing.
         """
+        logger.debug("Checking for availability of new ROR data")
         records_url = 'https://zenodo.org/api/communities/ror-data/records?sort=newest'
```

Comment:
Should this be a setting encase someone wants to point at a specific ROR file?

Reply:
Yes, good call. Added a setting.